catalogue/models/component/__init__.py

```diff
@@ -147,6 +147,61 @@ def __str__(self):
         return ""
 
 
+class Contact(models.Model):
+    # Contact Information
+    contact_person_surname = models.CharField(
+        "Vorname",
+        help_text="Vorname des zuständigen Mitarbeiters",
+        max_length=1000,
+    )
+    contact_person_lastname = models.CharField(
+        "Nachname",
+        help_text="Nachname des zuständigen Mitarbeiters",
+        max_length=1000,
+    )
+    contact_address_land = models.CharField(
+        "Land",
+        help_text="Land in welchem sich der Anbieter befindet",
```

Comment:
Should come with pre-defined choises or be removed. We currently focus on Germany anyway